crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -59,6 +61,14 @@ pub enum Error {
     #[capi(msg = "AQC library error")]
     Aqc,
 
+    /// Tried to poll an endpoint but nothing received yet.
+    #[capi(msg = "no response ready yet")]
+    Empty,
```

Comment:
i know this matches what AQC currently uses, but i think it might be more familiar as WouldBlock (or similar)